client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1075,6 +1076,7 @@
   "label.lead-time": "Months lead time",
   "label.leave-as-pending": "Leave as pending",
   "label.ledger": "Ledger",
+  "label.minimum-months-stock": "Minimum months of stock",
```

Comment:
I wonder if instead we should use `Max MOS` and `Min MOS` ... this doesn't look ideal:

<img width="217" height="78" alt="Screenshot 2025-09-22 at 9 28 27 AM" src="https://github.com/user-attachments/assets/27b90287-e990-4d7b-b57d-13127bf786ee" />